nwapp/utils/notifier.js

```diff
@@ -32,10 +35,23 @@ module.exports = function (spec) {
   if (!settings.showNotifications) return;
   playNotificationSound();
 
-  var notification = new window.Notification(spec.title, {
-    body: spec.message,
-    icon: getNotificationImageForUri(spec.link)
+  // We use this instead of `new window.Notification(...)` because we can disable the native sound
+  notifier.notify({
+    title: spec.title,
+    message: spec.message,
+    // absolute path
+    icon: getNotificationImageForUri(spec.link),
+    // Only Notification Center or Windows Toasters
+    // We handle the sound outside of this above
+    // And we set to false because Windows 10 by default makes a separate sound
+    sound: false,
+    // wait with callback until user action is taken on notification
+    wait: false
```

Comment:
@mikaelbr Thanks for the info. It seems to work just fine with it being false? I believed that option was to make the notification sit there until some user action as taken but that doesn't seem to be the case either.